packages/api/test/e2e/api/promise-contract.spec.ts

```diff
@@ -2,70 +2,91 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Address, Hash } from '@polkadot/types/interfaces';
-
 import fs from 'fs';
 import path from 'path';
-
 import { Abi } from '@polkadot/api-contract';
 import testingPairs from '@polkadot/keyring/testingPairs';
 import WsProvider from '@polkadot/rpc-provider/ws';
 
-import { SubmittableResult } from '../../../src';
-import ApiPromise from '../../../src/promise';
-import describeE2E from '../../util/describeE2E';
-
-import flipperAbi from '../../../../api-contract/test/contracts/flipper.json';
-const flipperCode = fs.readFileSync(path.join(__dirname, '../../../../api-contract/test/contracts//flipper-pruned.wasm')).toString('hex');
+import { Address, Hash } from '@polkadot/types/interfaces';
 
-describeE2E()('Promise e2e contracts', (wsUrl): void => {
+import { ApiPromise, SubmittableResult } from '../../../src';
+import { describeE2E } from '../../util';
+
+// Backwards compatibility:
+// A breaking change was introduced by substrate runtime spec version 97. https://github.com/paritytech/substrate/pull/2911/files
+// The change had to be implemented in ink! which changed the structure of the Wasm files.
+// The Polkadot JS API is only supporting substrate version greater or equal to spec_version 97.
+
+import incrementerAbi from '../../../../api-contract/test/contracts_97_/Incrementer.json'; // eslint-disable-line
```

Comment:
Locations.

Reply:
It's not in the `/src/` folder, so it's not part of the node module and I can't access it via `@polkadot/`

Reply:
Yes, I mean the `contracts_....`